src-ts/tools/learn/course-details/course-curriculum/CourseCurriculum.tsx

```diff
@@ -1,8 +1,8 @@
 import { Dispatch, FC, SetStateAction, useCallback, useState } from 'react'
 import { NavigateFunction, useNavigate } from 'react-router-dom'
 
-import { LearnCourse, LearnLesson, LearnModule } from '../../../../lib'
-import { TcAcademyPolicyModal } from '../../tc-academy-policy-modal'
+import { Button, LearnCourse, LearningHat, LearnLesson, LearnModule } from '../../../../lib'
```

Comment:
This is the same issue as i mentioned in my last PR about keeping the learn-specific components in the learn tool directory. So maybe instead of addressing this in each PR, we do a separate PR to clean it all up?